src/context/filter-context.js

```diff
@@ -1,5 +1,5 @@
 import {createContext} from 'react'
-import {useContext,useReducer} from 'react'
+import {useContext,useReducer,useState} from 'react'
```

Comment:
you can combine this into one line

Reply:
hey nikhil it is one line only updated useState